iquip/apps/scheduler.py

```diff
@@ -73,7 +78,7 @@ def run(self):
                             arguments=expid["arguments"]
                         ))
                     self.fetched.emit(schedule)
-        except WebSocketException:
+        except:  # pylint: disable=bare-except
```

Comment:
AFAIK, the bare except is in fact not desirable. Instead, `except Exception` would be better.
The bare except will prevent system level halts as well, for example Keyboard interrupt.

Reply:
Oh, I didn't know much about the bare except.
I will apply your suggestion.

Reply:
Umm.. Your solution causes a [broad-exception-caught](https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/broad-exception-caught.html).
Is it okay to disable this warning?